src/file_handler.py

```diff
@@ -3,7 +3,7 @@
 import gzip
 
 # function to open both gzip'd and regular files
-def gzopen(file_path, mode='rt', compresslevel=6):
+def gzopen(file_path, mode='rt', compresslevel=1):
```

Comment:
In the future releases, it might be a nice idea to make this _compresslevel_ an option for users to choose from.